<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QFrame" name="TitleBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(42, 46, 52);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(42, 46, 52);
border: 5px solid transparent;
border-image: url(:/Icon/Img/Icon_Gray/x.svg) 1 1 1 1  stretch stretch;
}
QPushButton:hover{
background-color: rgb(62, 68, 77);
}
QPushButton:pressed{
background-color: rgb(90, 99, 108);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="MinimizeButton">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(42, 46, 52);
border: 5px solid transparent;
border-image: url(:/Icon/Img/Icon_Gray/minus.svg) 1 1 1 1 stretch stretch;
}
QPushButton:hover{
background-color: rgb(62, 68, 77);
}
QPushButton:pressed{
background-color: rgb(90, 99, 108);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="DragBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;
font: 12pt &quot;KoPub돋움체 Bold&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>ER.GG</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="SideBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>190</width>
     <height>690</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(27, 31, 40);
border-right: 3px solid rgb(42, 46, 52);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;</string>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QRadioButton" name="TierListButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
text-align: center;
border-right: 3px solid rgb(116, 128, 139);
font: 9pt &quot;KoPub돋움체 Bold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(27, 31, 40);
}

QRadioButton:hover{
background-color: rgb(52, 57, 65);
}

QRadioButton:pressed{
background-color: rgb(61, 67, 76);
}

QRadioButton::indicator{
width: 25px;
height: 25px;
image: url(:/Icon/Img/Icon_Gray/bar-chart-2.svg);
}

QRadioButton::checked{
background-color: rgb(61, 67, 76);
}</string>
     </property>
     <property name="text">
      <string>Tier List</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="MatchRecordButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
font: 9pt &quot;KoPub돋움체 Bold&quot;;
border-right: 3px solid rgb(116, 128, 139);
color: rgb(255, 255, 255);
background-color: rgb(27, 31, 40);
}

QRadioButton:hover{
background-color: rgb(52, 57, 65);
}

QRadioButton:pressed{
background-color: rgb(61, 67, 76);
}
QRadioButton::indicator{
width: 25px;
height: 25px;
image: url(:/Icon/Img/Icon_Gray/align-left.svg)
}

QRadioButton::checked{
background-color: rgb(61, 67, 76);
}</string>
     </property>
     <property name="text">
      <string>Match Record</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="InGameMMRButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton{
font: 9pt &quot;KoPub돋움체 Bold&quot;;
border-right: 3px solid rgb(116, 128, 139);
color: rgb(255, 255, 255);
background-color: rgb(27, 31, 40);
}

QRadioButton:hover{
background-color: rgb(52, 57, 65);
}

QRadioButton:pressed{
background-color: rgb(61, 67, 76);
}
QRadioButton::indicator{
width: 25px;
height: 25px;
image: url(:/Icon/Img/Icon_Gray/search.svg)
}

QRadioButton::checked{
background-color: rgb(61, 67, 76);
}</string>
     </property>
     <property name="text">
      <string>In Game MMR</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="RightFrame">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>1090</width>
     <height>690</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(27, 31, 40);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="MainWindow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1070</width>
      <height>670</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(90, 99, 108);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="MainWindowPage1" native="true"/>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../UIResource/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
